comparison/config.go

```diff
@@ -37,6 +37,12 @@ type LoadTestConfig struct {
 	// This can also point to a local file if prefixed with "file://".
 	// In such case, the dump file will be uploaded to the app servers.
 	DBDumpURL string
+	// An optional list of IPs present in the posts from the DB dump
```

Comment:
How will the user get the list of IPs? Are they expected to go through the DB dump before and find out what IPs are there?

Reply:
My plan is that we (as in the server platform team) bundle all this information together, so devs only have to get a file that contains all this info and use some mechanism to feed that file and let the load-test take care of everything. For what it's worth, I've been getting these IPs by grepping the dump, so we can automate that as soon as the dump is generated.

Reply:
Any reason we can't do this as an internal step? We can grep the dump ourselves, get the IPs, and then replace them. We should try to avoid any manual steps as much as possible.

Reply:
Do you mean grepping the IPs during the load of the DB dump? I thought about that too, but it would be a bit redundant, since instead of doing it once for each DB dump (when generating it), we'd end up doing it each time we run a test that uses such dump. The time spent in grepping the IPs is not trivial either (I don't remember the specifics, but at least a minute if I'm not mistaken), so I'd rather get all this information when generating the dump and bundling it for the users, so it's transparent for them.
What we need is an automated process to generate the DB dumps, but this PR is the very first step to consume that data, assuming it's already created.
I'm open to change course here too, of course, so please feel free to push back on any of this.

Reply:
>  but this PR is the very first step to consume that data, assuming it's already created.

Ah okay, got you now. Yep, looking forward to the next step. :)